modules/reference/pages/software/software-channel-list-menu.adoc

```diff
@@ -1,47 +1,29 @@
 [[ref.webui.software.channel.list.menu]]
 = Channel List Menu
 
+The menu:Software[Channel List] section allows you to view and manage software channels and packages on your clients.
 
+For more information about software channels, see xref:client-configuration:channels.adoc[].
 
-[[ref.webui.software.channel.list.overview]]
-== Channels Overview
+The menu:Software[Channel List > All] section displays a list of all software channels that are available to your organization.
 
-The menu:Main Menu[Software > Channels List] menu item is the first to appear.
-A software channel provides packages grouped by products or applications to simplify the selection of packages to be installed on a system.
+Each software channel in the list shows a channel name, a provider, the number of packages and patches in the channel, and the number of clients currently subscribed to the channel.
+Click the plus sign next to the name of a parent channel to expand the entry and see all the related child channels.
+Click the channel name to see more information about the channel.
 
-There are two types of software channels: base channels and child channels.
+Within the menu:Software[Channel List] section you can select which subset of channels you would like to see by navigating to tabs, or the sub-menu items.
 
-[[s3-sm-channel-list-base]]
-Base Channels::
-A base channel consists of packages built for a specific architecture and release.
-For example, all of the packages in {sles}{nbsp}12 for the `x86_64` architecture make up a base channel.
-The list of packages in {sles}{nbsp}12 for the `s390x` architecture make up a different base channel.
-+
+[[channel-list-filters]]
+[cols="1,1,1", options="header"]
+.Channel List Filters
+|===
+| Filter | Description
+| All         | All channels available to your organization.
+| SUSE        | Channels provided by {suse}.
+| Popular     | Channels most subscribed to by clients in your organization.
+| My Channels | Software channels that belong to your organization, including custom channels.
```

Comment:
I think this and the description in the webui is confusing.  "belong" is too broad and these were previously known as custom channels (see the file name of the URL).  Thus these don't "include" but "are" custom channels.

Reply:
Yeah, when I edit the bit in the Admin Guide, I'm going to put more thought into this, I think, and will update this page accordingly.

Reply:
ok, makes sense.